qemu/tests/cfg/watchdog.cfg

```diff
@@ -41,8 +41,6 @@
                     watchdog_action = reset
                 - pause:
                     watchdog_action = pause
-                - poweroff:
-                    watchdog_action = poweroff
             variants:
```

Comment:
after watchdog triggered, guest will be poweroff, if migration not finished, this case will always failed.